src/main/java/frc/team2412/robot/util/autonomous/AutonomousChooser.java

```diff
@@ -97,50 +96,36 @@ private static SequentialCommandGroup getLineAutoCommand(Subsystems subsystems,
-                        subsystems.shooterSubsystem, subsystems.shooterVisionSubsystem, subsystems.drivebaseSubsystem,
-                        subsystems.intakeSubsystem),
-                "TWo ball auto",
-                Subsystems.SubsystemConstants.INDEX_ENABLED &&
-                        Subsystems.SubsystemConstants.SHOOTER_ENABLED &&
-                        Subsystems.SubsystemConstants.SHOOTER_VISION_ENABLED &&
-                        Subsystems.SubsystemConstants.DRIVE_ENABLED &&
-                        Subsystems.SubsystemConstants.INTAKE_ENABLED),
-        TWO_SCUFFED((subsystems, trajectories) -> new TwoBallScuffedAutoCommand(subsystems.indexSubsystem,
-                subsystems.shooterSubsystem, subsystems.shooterVisionSubsystem, subsystems.drivebaseSubsystem,
-                subsystems.intakeSubsystem), "TWO SCUFFED", true),
-        SQUARE_PATH((subsystems, trajectories) -> AutonomousChooser.getSquarePathAutoCommand(subsystems, trajectories),
-                "Square Path", Subsystems.SubsystemConstants.DRIVE_ENABLED),
-        LINE_PATH((subsystems, trajectories) -> AutonomousChooser.getLineAutoCommand(subsystems, trajectories),
-                "Line Path", Subsystems.SubsystemConstants.DRIVE_ENABLED),
-        STAR_PATH((subsystems, trajectories) -> AutonomousChooser.getStarPathAutoCommand(subsystems, trajectories),
-                "Star Path", Subsystems.SubsystemConstants.DRIVE_ENABLED),
-        WPI_PATH((subsystems, trajectories) -> AutonomousChooser.getAutoWPICommand(subsystems), "WPI Lib Path",
-                Subsystems.SubsystemConstants.DRIVE_ENABLED),
-        CLIMB((subsystems, trajectories) -> new ClimbTestCommand(subsystems.climbSubsystem), "Climb test",
-                Subsystems.SubsystemConstants.CLIMB_ENABLED),
-        INDEX((subsystems, trajectories) -> new IntakeInCommand(subsystems.indexSubsystem, subsystems.intakeSubsystem),
-                "Index test", Subsystems.SubsystemConstants.INDEX_ENABLED),
-        INTAKE((subsystems, trajectories) -> new IntakeTestCommand(subsystems.intakeSubsystem), "Intake test",
-                Subsystems.SubsystemConstants.INTAKE_ENABLED && Subsystems.SubsystemConstants.INDEX_ENABLED),
-        SHOOTER((subsystems, trajectories) -> new ShooterTurretSetAngleCommand(subsystems.shooterSubsystem,
-                subsystems.shooterSubsystem.getTurretTestAngle()), "Shooter test",
-                Subsystems.SubsystemConstants.SHOOTER_ENABLED),
-        INTAKE_SHOOTER(
-                (subsystems, trajectories) -> new FullShootCommand(subsystems.shooterSubsystem,
-                        subsystems.shooterVisionSubsystem, subsystems.intakeSubsystem, subsystems.indexSubsystem),
-                "Intake and shoot",
-                Subsystems.SubsystemConstants.INTAKE_ENABLED &&
-                        Subsystems.SubsystemConstants.INDEX_ENABLED &&
-                        Subsystems.SubsystemConstants.SHOOTER_ENABLED);
+        // spotless:off
```

Comment:
Please use the most recent formatting and then make the desired changes.

Reply:
confused on this, can fix later

Reply:
#123 fixed spotless's enum formatting, so we should probably keep the formatting.

Reply:
its fiiiine